android/titanium/src/java/org/appcelerator/titanium/view/TiDrawableReference.java

```diff
@@ -239,8 +248,15 @@ public static TiDrawableReference fromObject(KrollProxy proxy, Object object)
 			object = proxy.resolveUrl(null, (String) object);
 		}
 
+		boolean useCaches = false;
```

Comment:
Same as above, you can shrink this down to a one-liner

Reply:
fixed